packages/aws-cdk-lib/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -411,6 +411,12 @@ export interface CommonAutoScalingGroupProps {
    * @default false
    */
   readonly ssmSessionPermissions?: boolean;
+
+  /**
+   * The strategy for distributing instances across Availability Zones.
+   * @default CapacityDistributionStrategy.BALANCED_BEST_EFFORT
```

Comment:
this is the service default value, and not CDK default value. We should not mention the service default value here. I prefer mention that default is undefined